check/check.go

```diff
@@ -99,7 +105,7 @@ type Check struct {
 	ID             string           `yaml:"id" json:"test_number"`
 	Description    string           `json:"test_desc"`
 	Set            bool             `json:"omit"`
-	SubChecks      []*SubCheck       `yaml:"sub_checks"`
+	SubChecks      []*SubCheck      `yaml:"sub_checks"`
```

Comment:
NIT: Not in scope of this PR, but worth noting that we should avoid slices of pointers: https://medium.com/@philpearl/bad-go-slices-of-pointers-ed3c06b8bb41

Reply:
Good suggestion. I'm going to make changes later. This code is probably going to change as I start migrating from other *-bench projects into it.